snmp/datadog_checks/snmp/data/default_profiles/f5-big-ip.yaml

```diff
@@ -256,6 +256,27 @@ metrics:
           OID: 1.3.6.1.4.1.3375.2.2.10.1.2.1.1
           name: ltmVirtualServName
 
+  - MIB: F5-BIGIP-LOCAL-MIB
+    table:
+      OID: 1.3.6.1.4.1.3375.2.2.10.13.2
+      name: ltmVsStatusTable
+    symbols:
+      # Gauges
+      - name: ltmVsStatus
+        constant_value_one: true
+    metric_tags:
+      - tag: ltm_vs_status_avail_state
```

Comment:
1/ I think we also need to add the tag `ltmVsStatusName` (OID 1.3.6.1.4.1.3375.2.2.10.13.2.1.1):

```
metric_tags:
  - column:
      OID: 1.3.6.1.4.1.3375.2.2.10.13.2.1.1
      name: ltmVsStatusName
    tag: ltm_vs_status_name
```

Otherwise, user won't be able to know which status is enabled or not. Having `ltm_vs_status_name` tag will help with that. 

2/ You can probably also add `ltmVsStatusEnabledState` since if seems to be useful to know what is the "enabled state":

```
  - column:
      OID: 1.3.6.1.4.1.3375.2.2.10.13.2.1.3
      name: ltmVsStatusEnabledState
    tag: ltm_vs_status_enabled_state
``